src/System.ComponentModel.TypeConverter/tests/TypeDescriptorTests.cs

```diff
@@ -66,6 +67,22 @@ public void GetAssociationReturnsExpectedObject()
             Assert.Equal(secondaryObject, associatedObject);
         }
 
+        [Fact]
+        public void GetAssociationReturnsDesigner()
+        {
+            var designer = new MockDesigner();
+            var designerHost = new MockDesignerHost();
+            var component = new DescriptorTestComponent();
+
+            designerHost.AddDesigner(component, designer);
+            component.AddService(typeof(IDesignerHost), designerHost);
+
+            var associatedObject = TypeDescriptor.GetAssociation(designer.GetType(), component);
```

Comment:
Nit: var => whatever type this is (`object`?)

Reply:
Sure thing